photon-core/src/main/java/org/photonvision/vision/pipeline/AprilTagPipeline.java

```diff
@@ -87,7 +87,12 @@ protected void setPipeParamsImpl() {
         config.refineEdges = settings.refineEdges;
         config.quadSigma = (float) settings.blur;
         config.quadDecimate = settings.decimate;
-        aprilTagDetectionPipe.setParams(new AprilTagDetectionPipeParams(settings.tagFamily, config));
+
+        var quadParams = new AprilTagDetector.QuadThresholdParameters();
+        quadParams.minClusterPixels = 5;
```

Comment:
do we not wanna explicitly set the other parameters too?

Reply:
I'm good with setting them all. I did this one first to make sure it fixed the problem.

Reply:
Sweet, let's do that and just drop a comment explaining why we chose the numbers (they seem to work OK)